tests/unit/functions/webhooks/conftest.py

```diff
@@ -0,0 +1,126 @@
+import logging
+from unittest.mock import Mock, MagicMock, patch
+from requests import Response
+from aws_lambda_powertools.utilities.typing import LambdaContext
+
+import pytest
+
+logger = logging.getLogger(__name__)
+
+@pytest.fixture
+def mock_lambda_context():
+    context = LambdaContext()
+    context._function_name = "test"
+    context._memory_limit_in_mb = 128
+    context._invoked_function_arn = "XXXXXXXXXXXXXXXXX"
+    context._aws_request_id = "XXXXXXXXXXXXXXXXX"
+    return context
+
+@pytest.fixture
+def stub_source():
```

Comment:
This stub for a source looks like Q doing it's best to be "useful". But whilst it has some common fields with an actual TAMS Source it has a load of "extra stuff" and missing some "other stuff". Here is an example from the BBC's spec repo, not sure if the removal of fields will cause your tests issues though?
```suggestion
def stub_source():
    return {
        "id": "2aa143ac-0ab7-4d75-bc32-5c00c13d186f",
        "format": "urn:x-nmos:format:video",
        "label": "bbb",
        "description": "Big Buck Bunny video",
        "created_by": "tams-dev",
        "updated_by": "tams-dev",
        "created": "2008-05-27T18:51:00Z",
        "updated": "2008-05-27T18:51:00Z",
        "collected_by": [
            "86761f3a-5998-4cfe-9a89-8459bcb8ea52"
        ]
}
```